yaksh/templates/yaksh/course_detail.html

```diff
@@ -41,6 +41,19 @@
         <a href="{{URL_ROOT}}/exam/manage/send_mail/{{ course.id }}/">
         Send Mail</a>
         </li>
+        <li>
+            <form id="upload_users" action="{{ URL_ROOT }}/exam/manage/upload_users/{{course.id}}/"
+                method="POST" enctype="multipart/form-data">
+                {% csrf_token %}
+                <input type="file" name="csv_file" />
+                <button class="btn btn-primary" type=submit> Upload Users <span class="glyphicon glyphicon-open"/></button>
```

Comment:
Why break line!
I am using a div with particular class after button so no need to put br. Next element is already on next line.
Even if I put a break line it will show as it is.

Reply:
Exactly, intentions are not to separate out because the button and instructions look one piece.
So, user will definitely will read the instructions. You can once see UI wise, I am uploading the image. 
If not good then will add break.